### DISCUSSION TITLE
Digital Twin and Predictive Maintenance of Pneumatic Systems (Project 215)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#46] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 215](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Digital%20Twin%20and%20Predictive%20Maintenance%20of%20Pneumatic%20Systems)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by rohit4849** [comment]
Hello, 
I want to share this excellent master thesis report that recently got published by Chalmers University of Technology, Sweden. Students applied a data-driven approach to detect air leakage in a pneumatic system. They physically introduced defects in the machines and collected sensor data to train machine learning models and attained a great accuracy (98%-99%) for detecting leakage. 
However, in the proposed idea, you will get the opportunity to explore the possibility of using simulation tools like Simulink/Simscape to generate synthetic data for training Predictive Maintenance Models.    
https://odr.chalmers.se/bitstream/20.500.12380/302631/1/Johan%20Len%C3%A9%20and%20Mohan%20Rajashekarappa%20_MasterThesis_final_report.pdf

**Reply by of-sahin** [reply]
Dear friend I am getting error 403 with this link. Could you please share another?

**Reply by taoufikayoub** [reply]
@hawkulhawf try this link: https://odr.chalmers.se/handle/20.500.12380/302631

**Comment by GN-314** [comment]
Hi everyone ! (I'm so sorry for my bad English. French is my real language).

My name is Govy. I'm interested to that subject for my Master's thesis. I'll very happy to discuss about it with you.

**Reply by rohit4849** [reply]
Hi Govy, 
Thanks for the post. Nice to hear from you. Please go through some of the suggested literature and feel free to post any questions or suggestions you have, we will try to help you. There are a few others working on the same project. You can use this discussion forum to collaborate with them as well.  
Thanks,
Rohit

**Reply by GN-314** [reply]
Thank you so much dear Rohit !

**Comment by GN-314** [comment]
Hi ! Be welcome !
Yes, I'm still working on it. I'll very happy to work with you.

**Comment by GN-314** [comment]
That is my personal mail address : ***@***.***
We can talk about this subject.

**Comment by of-sahin** [comment]
Hi everyone I am also would like to work for DT and MPC.  My name is Omer Faruk. Don't be hesitate to contact me for this content.
of.sahin@outlook.com

**Comment by GN-314** [comment]
Hi Omer. I'll nice to collaborate with you on this project.

**Comment by amnforum** [comment]
Hi, I am a student at MSRIT and interested in working and collaborating on this project

**Reply by taoufikayoub** [reply]
Hi @amnforum , If you wanna collaborate feel free to contact me.
Here is my email: ayoub107p5@gmail.com

**Comment by AndySingh154** [comment]
Hi, I am Nidhi Rajput a student at The Maharaja Sayajirao University and interested in working and collaborating on this project.

**Comment by asma32074343** [comment]
Hello,I am currently preparing my master’s thesis in the field of Industry 4.0 and find this topic intriguing. I am considering it for my thesis but am still in the initial stages of defining my approach. I would appreciate any advice on this matter. Specifically, I am interested to know whether this topic requires any experimental work to be conducted.

**Comment by balajisai1211** [comment]
Hi! I'm a Balaji sai Tolapu student at Rutgers University, majoring in Industrial and Systems Engineering. I am keenly interested in collaborating on this project, particularly in areas involving MATLAB and Simulink for digital twin simulations. My academic and project experiences equip me to contribute effectively to your project's goals.

**Comment by balajisai121** [comment]
Hi! I'm a Balaji sai Tolapu student at Rutgers University, majoring in Industrial and Systems Engineering. I am keenly interested in collaborating on this project, particularly in areas involving MATLAB and Simulink for digital twin simulations. My academic and project experiences equip me to contribute effectively to your project's goals.

**Reply by Kimaniiiiiiiiiiii** [reply]
Hello

**Comment by Kimaniiiiiiiiiiii** [comment]
I'm working on a similar project using a digital twin but for centrifugal pumps.

**Comment by balajisai121** [comment]
hello

**Reply by Kimaniiiiiiiiiiii** [reply]
hello can we collaborate

**Comment by balajisai121** [comment]
Sure .
